### PR TITLE
feat: 🎸 add attention checks from task library

### PR DESCRIPTION
**Pull request by Rashi1997**

### CODE REVIEW

**Review by mcmcgrath13 — APPROVED** [review]
from a code perspective looks good (I assume you've run this and it looks as you expected) - I think @nicoleprovenza should have final say if things look right before merging

**Comment by Rashi1997** [comment]
@nicoleprovenza Can you check if this looks good in the electron, as expected

**Review by nicoleprovenza — APPROVED** [review]

**Comment by Rashi1997** [comment]
🚀

**Pull request merged**
